src/wilson/dom.cljs

```diff
@@ -3,21 +3,41 @@
   (:require [wilson.utils :refer [capitalize]]
             [clojure.string :as string]))
 
+(defn describe-key
+  [k]
+  (if (keyword? k)
+      (name k)
```

Comment:
Oh, I see; capitalization is still outside of the description. That should probably be moved in here; either done automatically for all descriptions, or manually for keywords here and then for prepare-keys vectors below; either way not in `table`